src/appShell/App/PortalHeader.tsx

```diff
@@ -43,7 +43,7 @@ export default class PortalHeader extends React.Component<
 
             {
                 id: 'tutorials',
-                text: 'Tutorials/Webinars',
+                text: 'Tutorials',
```

Comment:
Based on Aaron's description in the ticket, I think it's better to use `Learn` to replace 'Tutorials/Webinars' 
```suggestion
                text: 'Learn',
```